### ISSUE TITLE
Critical vulnerability (httpsnippet > event-stream > flatmap-stream)

### ISSUE THREAD

**Issue by JaneX8** [Kong/httpsnippet#121]
Please remove or replace depencency https://github.com/dominictarr/event-stream since it's issues (https://github.com/dominictarr/event-stream/issues) show the ridiculousness alone. Also 

> This repository has been archived by the owner. It is now read-only.

The reason for this crap is this critical vulnerability https://www.npmjs.com/advisories/737. 

```
  Critical        Malicious Package

  Package         flatmap-stream

  Patched in      No patch available

  Dependency of   httpsnippet

  Path            httpsnippet > event-stream > flatmap-stream

  More info       https://nodesecurity.io/advisories/737
```

**Comment by JaneX8** [comment]
I seem some related attempt by @darrenjennings here: https://github.com/Kong/httpsnippet/commit/cb28c9868e2ff47de41727a02e2d56bbbd66dd46 but version 1.16.6 is not published on npmjs.

**Comment by JaneX8** [comment]
Hi, @ahmadnassri. Can you please read the comment of darrenjennings here: https://github.com/Kong/httpsnippet/commit/cb28c9868e2ff47de41727a02e2d56bbbd66dd46.

**Comment by ahmadnassri** [comment]
https://github.com/Kong/httpsnippet/commit/cb28c9868e2ff47de41727a02e2d56bbbd66dd46#commitcomment-32031603

**Comment by darrenjennings** [comment]
Fixed in version [1.16.6](https://www.npmjs.com/package/httpsnippet/v/1.16.6)
